File: pizzeria-meal/src/main/java/pl/pizzeria/order/domain/Order.java
package pl.pizzeria.order.domain;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.io.Serializable;
import java.math.BigDecimal;
import java.util.Set;

@Getter
@AllArgsConstructor
@NoArgsConstructor
public class Order implements Serializable {
    private static final long serialVersionUID = -7322759631220603229L;

    private Set<OrderItem> orderItemSet;
    private OrderDetails orderDetails;
    private BigDecimal price;
}

File: angular-app/src/app/model/order/order.ts
import { OrderDetails } from "../order-details";
import { OrderItem } from "./order-item";

export class Order {
    id: string;
    orderItems: OrderItem[];
    orderDetails: OrderDetails;
    price: number;

    constructor(id: string, orderItems: OrderItem[], orderDetails: OrderDetails, price: number) {
        this.id = id;
        this.orderItems = orderItems;
        this.orderDetails = orderDetails;
        this.price = price;
    }
}
